### PR TITLE
[IPS-720] Update all references from Mobile Settings to Device Settings

### PR DESCRIPTION
**Pull request by asmclean**

### CODE REVIEW

File: articles/dashboard/guides/applications/enable-android-app-links.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ keytool -list -v -keystore my-release-key.keystore
 For more info about signing certificates, see Android's [Sign Your App](https://developer.android.com/studio/publish/app-signing.html) developer doc.
 :::
 
-![Add Mobile Settings](/media/articles/applications/mobile-settings.png)
+![Add Device Settings](/media/articles/applications/mobile-settings.png)
```
Comment (lbalmaceda) [inline articles/dashboard/guides/applications/enable-android-app-links.md]: This image referenced here is still showing the old name. Please update the resource following the steps [here](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#screenshots).

**Review by lbalmaceda — CHANGES_REQUESTED** [review]
Changes look good. Only one comment

File: articles/dashboard/guides/applications/enable-android-app-links.md (hunk 1 above)
Comment (asmclean) [reply articles/dashboard/guides/applications/enable-android-app-links.md]: Sorry, I should have left the PR in draft mode until I had a chance to get to that!

Images have been added.

**Review by lbalmaceda — APPROVED** [review]

**Comment by lbalmaceda** [comment]
Thanks!

**Pull request merged**